easybuild/easyconfigs/s/scVelo/scVelo-0.1.24-foss-2019b-Python-3.7.4.eb

```diff
@@ -0,0 +1,104 @@
+    ('Tkinter', '%(pyver)s'),
+    ('tqdm', '4.41.1'),
+]
+
+use_pip = True
+sanity_pip_check = True
+
+exts_default_options = {'source_urls': [PYPI_SOURCE]}
+
+exts_list = [
+    ('Cycler', '0.10.0', {
+        'modulename': 'cycler',
+        'source_tmpl': 'cycler-%(version)s.tar.gz',
+        'checksums': ['cd7b2d1018258d7247a71425e9f26463dfb444d411c39569972f4ce586b0c9d8'],
+    }),
+    ('kiwisolver', '1.1.0', {
+        'checksums': ['53eaed412477c836e1b9522c19858a8557d6e595077830146182225613b11a75'],
+    }),
+    ('matplotlib', '3.0.3', {
+        'prebuildopts': "export CPLUS_INCLUDE_PATH=$EBROOTFREETYPE/include/freetype2:${CPLUS_INCLUDE_PATH} && ",
+        'preinstallopts': "export CPLUS_INCLUDE_PATH=$EBROOTFREETYPE/include/freetype2:${CPLUS_INCLUDE_PATH} && ",
+        'source_urls': ['https://pypi.python.org/packages/source/m/matplotlib'],
+        'checksums': ['e1d33589e32f482d0a7d1957bf473d43341115d40d33f578dad44432e47df7b7'],
+    }),
+    ('numpy-groupies', '0.9.10', {
+        'modulename': 'numpy_groupies',
+        'source_tmpl': 'numpy_groupies-%(version)s.tar.gz',
+        'checksums': ['a6835c252b7bd489e0f701f8bf5bc880c647ef251e3b434998b95a8aecc9b68a'],
+    }),
+    ('seaborn', '0.9.0', {
+        'checksums': ['76c83f794ca320fb6b23a7c6192d5e185a5fcf4758966a0c0a54baee46d41e2f'],
+    }),
+    ('natsort', '7.0.0', {
+        'checksums': ['7207cddc510e1d5e728a7f73a0230f20a65e3346acd54856be6252291690d3ec'],
+    }),
+    ('get_version', '2.1', {
+        'use_pip': False,
+        'checksums': ['8156b526c2557537b8ca82241fa2b82b3da25939627398f6567dee31ba9725bc'],
+    }),
+    ('legacy-api-wrap', '1.2', {
+        'use_pip': False,
+        'checksums': ['034a44612da7e9943d3964363a98937ab54d55e3301075374abe0d521eb8101b'],
+    }),
+    ('loompy', '3.0.6', {
+        'checksums': ['58e9763b8ab1af2a4a0e3805d120458b5184fd2b0f3031657ecce33c63ca4c46'],
+    }),
+    ('anndata', '0.7.1', {
+        'checksums': ['0940eb6e95a96469738120fd498ecd30904308f995b429e20a2ac852f25dce66'],
+    }),
+    ('scanpy', '1.4.5.post2', {
+        'patches': ['scanpy-1.4.5.post2_fix_h5py_version.patch'],
+        'checksums': [
+            'e36d498ac53e21aa0de9dada579ab98570c2e8552b29206eedd0aaeac91eec96',  # scanpy-1.4.5.post2.tar.gz
+            # scanpy-1.4.5.post2_fix_h5py_version.patch
+            '6989a2e5cb60c76d5f025c9daa02085737b8480e8bf4847f7412a5a96ccc833b',
+        ],
+    }),
+    ('umap-learn', '0.3.10', {
+        'modulename': 'umap',
+        'checksums': ['21ce6b6d7486905318ba6458b5a9ba2cfb935878d30c24e6fba64ee3bd504d09'],
+    }),
+    (name, version, {
+        'modulename': 'scvelo',
+        'source_tmpl': '%(namelower)s-%(version)s.tar.gz',
+        'checksums': ['a9e1842ec6a55c982a853349238a494464c447e208aa5d46d3a7d225a910113f'],
+    }),
+]
+
+sanity_check_commands = [
+    """python -c 'import matplotlib; matplotlib.use("TkAgg"); import matplotlib.pyplot' """,
+    "python -c 'from mpl_toolkits.mplot3d import Axes3D'",
```

Comment:
was this copied from the matplotlib easyconfig?

Reply:
Yes (but see my comment re the matplotlib version)

Reply:
Fixed in 393b5d6